cloud-sql/mysql/servlet/pom.xml

```diff
@@ -64,14 +64,24 @@
       <artifactId>HikariCP</artifactId>
       <version>3.4.1</version>
     </dependency>
+    <dependency>
+       <groupId>org.slf4j</groupId>
```

Comment:
Why are these added?

Reply:
There was an annoying logging warning about slf4j not being found. Though it doesn't break anything it was distracting. I can remove if needed.

Reply:
LGTM, was just curious.